frontend/app/src/components/history/transactions/TransactionContent.vue

```diff
@@ -73,15 +73,9 @@
         :class="$style.table"
         @update:options="updatePaginationHandler($event)"
       >
-        <template #item.ignoredInAccounting="{ item, isMobile }">
+        <template #item.ignoredInAccounting="{ item }">
           <div v-if="item.ignoredInAccounting" class="pl-4">
-            <badge-display v-if="isMobile" color="grey">
-              <v-icon small> mdi-eye-off </v-icon>
-              <span class="ml-2">
-                {{ tc('common.ignored_in_accounting') }}
-              </span>
-            </badge-display>
```

Comment:
How does this look on mobile now? is there any reason for the removal?

Reply:
Actually I removed the mobile view for this table, because it was too chaotic with default mobile view from vuetify.

Reply:
But I think it's better to not remove it, in case we will work on the mobile view of this table.

Reply:
Yes, can you improve it instead of removing it? I can think at least of one user that is using rotki on mobile, they might be more.

Reply:
Updated